dist/Carp/README

```diff
@@ -7,7 +7,7 @@ DESCRIPTION
 The Carp routines are useful in your own modules because they act like
 die() or warn(), but with a message which is more likely to be useful
 to a user of your module.  In the case of cluck, confess, and longmess
-that context is a summary of every call in the call-stack.  For a shorter
+that context is a summary of every call in the call stack.  For a shorter
```

Comment:
I would prefer if you leave this change out of this pull request.   We have 3 variants in the core distribution.
```
$ ack -l 'call(\s+|-)?stack' . | wc -l
28
```
Since I suspect that each variant will have its defenders, I recommend keeping that discussion out of this ticket so that we can focus on things that are more clearly erroneous.